### DISCUSSION TITLE
Document common integrations' basics

### DISCUSSION THREAD

**Discussion by peter-csala** [App-vNext/Polly#2199] [Ideas]
Hi folks,

Here at the discussions space and on StackOverflow I can see recently a recurring theme _"how to integrate Polly with XYZ?"_ These integrations are varying from HttpClient, RestSharp through FluentResult, OneOf till Entity Framework Core, grpc.net...

What do you guys would it make sense to have a dedicated _common integrations_ section on pollydocs where we demonstrate some basic use cases?

cc: @martincostello , @martintmk

**Comment by martincostello** [comment]
Sounds reasonable to me, though I would just link off to other docs (e.g. in the `HttpClient` case for HttpClientFactory and the new resilience libraries) if there's existing content rather than creating our own.
